src/test/groovy/tests/RepositoryTest.groovy

```diff
@@ -24,40 +24,27 @@ import static org.hamcrest.Matchers.hasSize
 
 class RepositoryTest extends RepositorySteps{
     Yaml yaml = new Yaml()
-    def configFile = new File("./src/test/resources/testenv.yaml")
     def repoListHA = new File("./src/test/resources/repositories/CreateDefault.yaml")
     def repoListJCR = new File("./src/test/resources/repositories/CreateJCR.yaml")
     def artifact = new File("./src/test/resources/repositories/artifact.zip")
-    def config = yaml.load(configFile.text)
     def utils = new Utils()
     def artifactoryURL
-    def protocol
-    def artifactoryBaseURL
-    def dockerURL
-    def username
-    def password
 
     @BeforeTest(groups=["jcr", "pro", "docker"])
     def setUp() {
-        artifactoryURL = config.artifactory.external_ip
-        dockerURL = config.artifactory.url
-        username = config.artifactory.rt_username
-        password = config.artifactory.rt_password
-        protocol = config.artifactory.protocol
-        artifactoryBaseURL = "${protocol}${config.artifactory.external_ip}/artifactory"
-        //RestAssured.baseURI = "http://${artifactoryURL}/artifactory"
+        artifactoryURL = "${artifactoryBaseURL}/artifactory"
```

Comment:
This is also a constant for the life of this class - set it as a member variable and be done - remove the function

Reply:
I've removed the base URL declaration, but RestAssured authentication options should be scoped only to this particular test.